azure-pipelines.yml

```diff
@@ -24,67 +40,80 @@ phases:
 
   - script: |  
       echo run build script
-      mkdir -p build_output
       set PATH=%PATH%;C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\MSBuild\15.0\Bin\
-      ./build.cmd Test
+      ./build.cmd Full
     displayName: 'Build Fabulous'
-  - task: PublishTestResults@1
-    inputs:
-      testResultsFormat: 'VSTest' # Options: JUnit, NUnit, VSTest, xUnit
-      testResultsFiles: 'build_output\testresults\*.xml' 
+
   - task: CopyFiles@2
     inputs:
       contents: build_output\*.nupkg
       targetFolder: $(Build.ArtifactStagingDirectory)
+
   - task: PublishBuildArtifacts@1
     inputs:
       pathtoPublish: $(Build.ArtifactStagingDirectory)
       artifactName: Windows-FabulousBuildOutputs
 
-
-- phase: macOS
-  displayName: macOS
-
-  condition: succeeded()
-  queue:
-    name: Hosted macOS
+- job: macOS
+  displayName: 'macOS (Complete)'
+  pool:
+    vmImage: 'macOS-10.13'
   steps:
   - script: |
         echo install fake
         export DOTNET_ROOT=$HOME/.dotnet/
-        export PATH=$PATH:$HOME/.dotnet/tools:/Library/Frameworks/Mono.framework/Versions/Current/Commands
         dotnet tool install fake-cli -g
+    displayName: 'Install Fake'
 
+  - script: |
         echo run build script
-        mkdir -p build_output
-        echo $PATH
-        ./build.sh Test
+        export DOTNET_ROOT=$HOME/.dotnet/
+        export PATH=$PATH:$HOME/.dotnet/tools:/Library/Frameworks/Mono.framework/Versions/Current/Commands
+        ./build.sh Full
     displayName: 'Build Fabulous'
-  - task: PublishTestResults@1
-    inputs:
-      testResultsFormat: 'VSTest' # Options: JUnit, NUnit, VSTest, xUnit
-      testResultsFiles: 'build_output/testresults/*.xml' 
+
   - task: CopyFiles@1
     inputs:
-      contents: build_output/*.nupkg
+      contents: ./build_output/*.nupkg
       targetFolder: $(Build.ArtifactStagingDirectory)
+
   - task: PublishBuildArtifacts@1
     inputs:
       pathtoPublish: $(Build.ArtifactStagingDirectory)
       artifactName: macOS-FabulousBuildOutputs
 
-# - phase: Linux
-#   displayName: Linux
-
-#   condition: succeeded()
-#   queue:
-#     name: Hosted Ubuntu 1604
-#   steps:
-#   - script: |
-#         echo install fake
-#         export PATH=$PATH:$HOME/.dotnet/tools:/Library/Frameworks/Mono.framework/Versions/Current/Commands
-#         dotnet tool install fake-cli -g
-#         echo run build script
-#         mkdir -p build_output
-#         ./build.sh Test
-#     displayName: 'Build Fabulous'
+- job: Linux
+  displayName: 'Linux (Complete)'
+  pool:
+    vmImage: 'ubuntu-16.04'
+  steps:
+  - script: |
+        echo install fake
+        export DOTNET_ROOT=/usr/share/dotnet
+        dotnet tool install fake-cli -g
+    displayName: 'Install Fake'
+
+  - script: |
```

Comment:
I am just wondering why this is necessary. According to the [docs ](https://github.com/Microsoft/azure-pipelines-image-generation/blob/master/images/linux/Ubuntu1604-README.md) this should already be installed?

Reply:
Yes, that was a desperate attempt at fixing msbuild. :)
I think the build can be fixed by finding the real path of msbuild (with support for .NET Standard)

Reply:
I know exactly what you mean 😅 It is the exact same error I had back when I first tried it. Perhaps it uses the wrong msbuild from mono. I will give it a spin in a local Linux VM.